lib/basisu/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+# Copyright 2017-2020 The Khronos Group Inc.
+# SPDX-License-Identifier: Apache-2.0
+
 project(basisu)
 
 cmake_minimum_required(VERSION 3.0)
```

Comment:
Please make a separate PR for the changes to `lib/basisu`. I need to pull a very large change from Binomial soon and don't want the merge nightmare. I prefer to pull the change then apply a PR for adding SPDX, etc. I might even try to get Binomial to REUSEIFY their repo.

Reply:
Done. I asked in https://github.com/BinomialLLC/basis_universal/issues/165
 about REUSE for their repo.